src/components/try-hcp-callout/docs.mdx

```diff
@@ -0,0 +1,41 @@
+---
+componentName: 'TryHcpCallout'
+---
+
+The `<TryHcpCallout />` components render HCP themed callout cards. There are multiple different components to meet various use cases, each composed of similar pieces.
+
+## Components
+
+### `<TryHcpCallout />`
+
+This composition includes a mock UI visual on large viewports. Intended for use in main content areas, such as on product landing pages.
+
+<LiveComponent>{`
+<TryHcpCallout
+  productSlug="terraform"
+	heading="Terraform Cloud"
+	description="Automate infrastructure provisioning at any scale for free."
+	ctaText="Try Cloud for free"
+	ctaUrl="https://app.terraform.io/public/signup/account"
+/>
+`}</LiveComponent>
+
+### `<TryHcpCalloutCompact />`
+
+This composition includes logo and text elements only. Intended for use in width-constrained areas, such as the sidecar area of our sidebar-sidecar layout.
```

Comment:
Do we have guidance from design yet on which pages / places we are adding this? Is that planned for a follow up PR?

Reply:
Yep, planning a follow-up PR! As far as I know, only placements are on the `/<product>` landing in main content area ("main" version of the callout), & `/<product>/downloads` views in the sidecar ("compact" version of the callout).